src/onnx/parse_resize.cpp

```diff
@@ -97,22 +97,19 @@ const auto& get_original_idx_op(const std::string& mode)
 static std::vector<int>
 calc_neighbor_points(const std::vector<std::vector<std::vector<std::size_t>>>& vvv_ind,
                      int i_dim,
-                     const std::vector<std::vector<std::size_t>>& vec_dims,
+                     std::vector<std::vector<std::size_t>>& vec_dims,
```

Comment:
I would prefer this to be either by value or rvalue reference, and use `std::move` to pass the vector. This will make it clear at the call site that the vector wont be used anymore.

Reply:
Done.